ikasaneip/pom.xml

```diff
@@ -710,7 +710,6 @@
                 <module>recovery-manager</module>
```

Comment:
Arent there more solr related libs included as dependencies in this pom

Reply:
I'll take a look